core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -402,6 +400,29 @@
             + "FROM \"SCOTT\".\"DEPT\") AS \"t1\" ON \"t0\".\"DEPTNO\" = \"t1\".\"DEPTNO\"");
   }
 
+  @Test void testJoinConditionAlwaysTruePushDown() {
+    CalciteAssert.model(JdbcTest.SCOTT_MODEL)
+        .query(
+            "select empno, ename, d.deptno, dname\n"
+                + "from scott.emp e,scott.dept d\n"
+                + "where true")
+        .explainContains(
```

Comment:
please match the formatting of strings in other tests. double quotes right after open parentheses.